wdt_port_block_test.py

```diff
@@ -35,10 +35,19 @@ def main():
                                         stdout=subprocess.PIPE)
 
     connection_url = receiver_process.stdout.readline().strip()
-    url_match = re.search('\?.*ports=([0-9]+).*', connection_url)
-    rest_of_url = url_match.group(0)
-    port_to_block = url_match.group(1)
-
+    print(connection_url)
+    url_match = re.search('\?(.*&)?ports=([0-9]+).*', connection_url)
```

Comment:
Let's comment we expect the url to look like either xxx or yyy with examples of each